aws/v20.0.0/release.yaml

```diff
@@ -49,10 +49,10 @@ spec:
     releaseOperatorDeploy: true
     version: 0.1.1
   - catalog: control-plane-catalog
-    name: capa-iam-controller
-    reference: 0.2.0
+    name: capa-iam-operator
+    reference: 0.3.0
```

Comment:
I think we do not need the reference anymore

Reply:
I wasn't sure what `reference` was used for so I just followed what was already there. If it's safe to remove I can do that.

Reply:
it's also fine leaving it right now, no biggie :)